views/update/update-director-check-answers.html

```diff
@@ -18,6 +18,9 @@
       {% include "includes/list/errors.html" %}
       
       <form action="" method="post">
+        <span class="govuk-caption-l gem-c-title__context">
+          {{name}}
```

Comment:
is this leftover from testing?

Reply:
No the prototype had the directors name at the top of the page so this was to add that - reusing the existing variable "name" which did the formatting etc